src/pages/profile.js

```diff
@@ -6,20 +7,43 @@ import ProfileBadge from '../components/profile-badge/profile-badge';
 import ProfileContent from '../components/profile-content/profile-content';
 import ProfileFilterMenu from '../components/profile-filter-menu/profile-filter-menu';
 import GitHubIcon from '-!svg-react-loader!../images/github.svg';
+import DisqusIcon from '-!svg-react-loader!../images/disqusIcon.svg';
 import { useAuth0 } from '@auth0/auth0-react';
+import { GetUser } from '../services/apiService';
 
 const Profile = ({ data, gitHubUsername }) => {
   const [selectedFilter, setSelectedFilter] = useState(4);
   const [listChange, setListChange] = useState(0);
-  const { user, isAuthenticated, loginWithRedirect } = useAuth0();
+  const {
+    user,
+    isAuthenticated,
+    loginWithRedirect,
+    getIdTokenClaims,
+  } = useAuth0();
+  const [commentsConnected, setCommentsConnected] = useState(false);
   const [bookmarkedRulesCount, setBookmarkedRulesCount] = useState();
   const [superLikedRulesCount, setSuperLikedRulesCount] = useState();
   const [likedRulesCount, setLikedRulesCount] = useState();
   const [dislikedRulesCount, setDislikedRulesCount] = useState();
   const [superDislikedRulesCount, setSuperDislikedRulesCount] = useState();
   const [commentedRulesCount, setCommentedRulesCount] = useState();
 
-  useEffect(() => {}, [listChange, user]);
+  async function CheckUser() {
+    const jwt = await getIdTokenClaims();
+    GetUser(user.sub, jwt.__raw)
+      .then((success) => {
+        setCommentsConnected(success.commentsConnected);
+      })
+      .catch((error) => {
+        console.error(error);
+      });
```

Comment:
This seems very odd. Is this an existing standard? I would think we don't want to be logging our errors to the console because then anyone can see them if they open the debugger console. Also if there is an error, shouldn't we display some kind of error message or something?